azurelinuxagent/common/protocol/wire.py

```diff
@@ -147,14 +157,14 @@ def get_vmagent_pkgs(self, vmagent_manifest):
         valid_pkg_list = self.client.filter_package_list(vmagent_manifest.family, ga_manifest, goal_state)
         return valid_pkg_list
 
-    def get_ext_handlers(self):
+    def get_ext_conf(self):
```

Comment:
WireProtocol.get_ext_handlers is an override of Protocol.get_ext_handlers. This change would need to happen across the whole class hierarchy (Protocol and MetadataProtocol need to be updated)

Reply:
Moved across hierarchy